src/Spout/Writer/Style/BorderBuilder.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Box\Spout\Writer\Style;
+
+/**
+ * Class BorderBuilder
+ */
+class BorderBuilder
+{
+    /**
+     * @var Border
+     */
+    protected $border;
+
+    public function __construct()
+    {
+        $this->border = new Border();
+    }
+
+    /**
+     * @param string $style Border style @see BorderPart::allowedStyles
+     * @param string $color Border A RGB color code
+     * @param string $width Border width @see BorderPart::allowedWidths
+     * @return self
+     */
+    public function setBorderTop($style = Border::STYLE_SOLID, $color = Color::BLACK, $width = Border::WIDTH_THICK)
```

Comment:
When using default values, we need to be careful of the ordering. Values that are most likely to be specified should go first and values that will be specified less often should go last. This is to prevent people from always having to specify the first values when all they want to do is set the last value.

In this case, I'd say that the order should be: color > width > style (or eventually width > color > style). So I'd move params around to reflect that. What do you think about the order?